x-pack/plugins/infra/server/lib/domains/log_entries_domain/builtin_rules/filebeat_system.ts

```diff
@@ -10,6 +10,15 @@ export const filebeatSystemRules = [
       exists: ['system.syslog.message'],
     },
     format: [
+      {
+        constant: '[Syslog][system] ',
```

Comment:
Given that it's the system module which contains both syslog and auth, how about

```suggestion
        constant: '[System][syslog] ',
```

?